### PR TITLE
 Add docker integration tests to our integration testing suite

### PR DESCRIPTION
**Pull request by mhagen-vmware**
This PR pulls the full docker integration tests in our CI environment so that they will be run every time now.

### CODE REVIEW

File: tests/test-cases/Group4-Docker-Integration/4-1-Docker-Integration.robot [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+Documentation  Test 4-1 - Docker Integration
+Resource  ../../resources/Util.robot
+Suite Setup  Install VIC Appliance To Test Server  false
+Suite Teardown  Cleanup VIC Appliance On Test Server
+
+*** Test Cases *** 
+Docker Integration Tests
+    Log To Console  \nStarting Docker integration tests...
+    Set Environment Variable  GOPATH  /go
+    ${out}=  Run Process  DOCKER_HOST\=tcp://${params} go test  shell=True  cwd=/go/src/github.com/docker/docker/integration-cli
```
Comment (andrewtchin) [inline tests/test-cases/Group4-Docker-Integration/4-1-Docker-Integration.robot]: Do we `go get github.com/docker/docker/integration-cli` in another step?

Comment (hagenmt) [inline tests/test-cases/Group4-Docker-Integration/4-1-Docker-Integration.robot]: It is in the base image

Comment (andrewtchin) [inline tests/test-cases/Group4-Docker-Integration/4-1-Docker-Integration.robot]: ✅

**Comment by andrewtchin** [comment]
lgtm

**Comment by mdubya66** [comment]
lgtm

**Comment by matthewavery** [comment]
LGTM

**Pull request merged**